### ISSUE TITLE
Accessibility: Landing page is missing some landmarks including `banner`, `main`, and `nav` in `footer`

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1045]
## Issue
The Sphinx Community landing page is missing some landmarks that enhance screen reader navigation. 

## Expected Behavior
- [WCAG 2.2 A 1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html) requires that information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.
- Screen reader users expect to hear structural information that will help them navigate a page including important landmark regions.
## Acceptance Criteria
- [x] Review [Scott O'Hara's Accessible Landmarks](https://www.scottohara.me/blog/2018/03/03/landmarks.html) blog post
- [x] Add `banner`, `main`, and `nav` in `footer` to the homepage
- [x] Ensure these elements are properly structured and nested e.g. https://www.html5accessibility.com/tests/roles-land.html 
- [x] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. - ensure the appropriate landmark regions are announced (e.g. Use Insert + F7 in NVDA)

_Linked to tracking issue: #456_

**Comment by jason-me** [comment]
Pending closure via PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140

**Comment by jason-me** [comment]
Merged in PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140
